sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
+    if not array or len(array) <= 1:
+        return array # nothing to sort
```

Comment:
while it works fine here, we might consider moving the comment to above line 3 to make it easier to understand as we read downwards.